cfgov/jinja2/v1/ask-cfpb/answer-page.html

```diff
@@ -124,17 +124,18 @@ <h2 class="a-heading">
                         Related {{category.name}} subjects
                     </h2>
                 </header>
-                <ul class="list list__unstyled list__links">
+                <ul class="m-list m-list__unstyled m-list__links">
                     {% for subcat in subcategories %}
-                        <li class="list_item">
-
-                            <a href="{{slugurl('category-' + subcat.parent.slug)}}{{subcat.slug}}" class="list_link">{{ subcat.name }}</a>
+                        <li class="m-list_item">
+                            <a href="{{slugurl('category-' + subcat.parent.slug)}}{{subcat.slug}}" class="m-list_link">
+                              {{ subcat.name }}
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/4355bc388be89b6e7a6a2ce200c793f12b3051de